auditlog/views/auditlog_view.xml

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <menuitem
+        id="menu_audit"
+        name="Audit"
+        parent="base.menu_custom"
+        sequence="50"
+        groups="base.group_system"
+    />
+    <!-- auditlog.rule -->
+    <record model="ir.ui.view" id="view_auditlog_rule_form">
+        <field name="name">auditlog.rule.form</field>
+        <field name="model">auditlog.rule</field>
+        <field name="arch" type="xml">
+            <form string="Rule">
+                <header>
+                    <button
+                        string="Subscribe"
+                        name="subscribe"
+                        type="object"
+                        states="draft"
+                        class="oe_highlight"
+                    />
+                    <button
+                        string="Unsubscribe"
+                        name="unsubscribe"
+                        type="object"
+                        states="subscribed"
+                    />
+                    <field name="state" widget="statusbar" />
+                </header>
+                <sheet>
+                    <group string="Rule">
+                        <group colspan="1">
+                            <field name="name" required="1" />
+                            <field name="model_id" />
+                            <field name="log_type" />
+                            <field
+                                name="action_id"
+                                readonly="1"
+                                groups="base.group_no_one"
+                            />
+                        </group>
+                        <group colspan="1">
+                            <field name="log_read" />
+                            <field name="log_write" />
+                            <field name="log_unlink" />
+                            <field name="log_create" />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_auditlog_rule_tree">
+        <field name="name">auditlog.rule.tree</field>
+        <field name="model">auditlog.rule</field>
+        <field name="arch" type="xml">
+            <tree
+                decoration-info="state == 'draft'"
+                decoration-bf="state == 'subscribed'"
```

Comment:
```suggestion
```

I would remove this decoration to align to other decorations across odoo.
Normally, "validated/posted" records have no decoration.